### PR TITLE
Add "The Curse of the A-word" blog post

### PR DESCRIPTION
**Pull request by sergeybykov**
Added "The Curse of the A-word" blog post.

### CODE REVIEW

File: blog/2021-05-05-the-curse-of-the-a-word.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+In my opinion, this rule simply means that an actor can change its internal state, whether it’s a formal state machine or a boolean/enum flag that will define how the actor should process another call. For example, Digital Twins are a mainstream pattern to model program entities that shadow physical IoT devices in order to reflect their state and to communicate with them. Actors are an obvious fit to implement Digital Twins.
+
+When a Digital Twin actor receives a "turn device off" command, it is very natural for the actor to flip an internal state variable that reflects the “off” state. In that state, the actor ignores or rejects all commands except for a "turn device on", which flips that variable back to “on”.
+
+## Elephant in the Room: Actors & Models
+
+![Actor Model](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/hgwatx1rr8zxfamkbnx0.png)
```
Comment (sw-yx) [inline blog/2021-05-05-the-curse-of-the-a-word.md:78]: feedback - meme slide a bit distracting. personally i would drop it. i dont feel strongly about this, but just mentioning

File: blog/2021-05-05-the-curse-of-the-a-word.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+author: Sergey Bykov
+author_title: Engineering
+author_image_url: https://avatars2.githubusercontent.com/u/8248806?s=460&v=4
+image: https://avatars2.githubusercontent.com/u/8248806?s=460&v=4
+release_version: V1.9.1
+---
+
+<!--truncate-->
+
+![Actors](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/rlw56fpp0hjgd73xmq8h.png)
```
Comment (sw-yx) [inline blog/2021-05-05-the-curse-of-the-a-word.md:20]: suggest removing, bit broey

File: blog/2021-05-05-the-curse-of-the-a-word.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+I wanted to write this post ever since I saw David Fowler's tweet and the discussion it triggered.
+![David Fowler](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/qt8irq2n337p5tsf7u4i.png)
+While it may have taken me quite some time to get around to answering it, 2020 wasn't an ordinary year by any measure.
```
Comment (sw-yx) [inline blog/2021-05-05-the-curse-of-the-a-word.md:24]: - your opening para is what people use to decide if they will read the rest
- i dont know who david fowler is. most people outside .net may not know who he is.
- using screenshot of tweet is inaccessible for screen readers
- mentioning 2020 is a reason, yes, but is it important enough to put in the first para? is there some better hook to get interest?

```suggestion

**Why aren’t actor frameworks more popular?**

I've wanted to write this post ever since [David Fowler's tweet](https://twitter.com/davidfowl/status/1189031080076009472?lang=en) and the discussion it triggered.

In short: Naming it "actors" caused a minefield of conflations, and both Microsoft Orleans and Cloudflare Workers have avoided them by actively avoiding the word. You should too.
```

this more directly answers the question posed by the blog title and invites the reader to read on.

just a suggestion

**Review by sw-yx — APPROVED** [review]
my 2 cents here, all ignorable but just offering
